ssl/d1_lib.c

```diff
@@ -1088,3 +1091,39 @@ unsigned int dtls1_min_mtu(SSL *s)
 {
     return dtls1_link_min_mtu() - BIO_dgram_get_mtu_overhead(SSL_get_wbio(s));
 }
+
+static unsigned int dtls1_get_data_mtu(SSL *s)
+{
+	unsigned int mac_overhead, int_overhead, blocksize, ext_overhead;
+	const SSL_CIPHER *ciph = SSL_get_current_cipher(s);
+	unsigned mtu = s->d1->mtu;
+
+	if (ciph == NULL)
+		return 0;
+
+	if (!ssl_cipher_get_overhead(ciph, &mac_overhead, &int_overhead,
+				     &blocksize, &ext_overhead))
+		return 0;
+
+	if (SSL_USE_ETM(s))
+		ext_overhead += mac_overhead;
+	else
+		int_overhead += mac_overhead;
+
+	/* Subtract external overhead (e.g. IV/nonce, separate MAC) */
+	if (ext_overhead >= mtu)
+		return 0;
+
+	mtu -= ext_overhead + DTLS1_RT_HEADER_LENGTH;
```

Comment:
Good spot.  The testing showed me I forget the header but then in fixing it I forgot the guard. Will fix.